tests/model/test_model.py

```diff
@@ -783,6 +783,36 @@ def test__handle_message_event(self, mocker, user_profile, response,
         # LOG REMAINS THE SAME IF UPDATE IS FALSE
         assert model.msg_list.log == log
 
+    @pytest.mark.parametrize('found_newest', [
+        (True),
```

Comment:
For a one-parameter entry, this is just `True`, ie. the parentheses are unnecessary.